runtime/development/src/lib.rs

```diff
@@ -707,7 +713,7 @@ parameter_types! {
 	pub const LaunchPeriod: BlockNumber = 7 * DAYS;
 	pub const VotingPeriod: BlockNumber = 7 * DAYS;
 	pub const FastTrackVotingPeriod: BlockNumber = 3 * HOURS;
-	pub const InstantAllowed: bool = false;
+	pub const InstantAllowed: bool = true;
```

Comment:
Are we OK with this for our development runtime? Otherwise, I'm not sure how to speed up the current integration test that goes through democracy.

Reply:
Ah, sure. Thanks for this!